src/main/java/TaskList.java

```diff
@@ -0,0 +1,62 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private final String SEP = "____________________________________________________________";
+    private final List<Task> TASKS;
+
+    public TaskList() {
+        this.TASKS = new ArrayList<>();
+    }
+
+    public void addTask(Task task) {
+        TASKS.add(task);
+    }
+
+    public void printTasks() {
+        System.out.println("Todo List:");
+        int counter = 1;
+        for (Task t: TASKS) {
```

Comment:
Consider using a traditional for-loop standard since you are using a counter anyways. This can help the code to be more readable and perhaps avoid further issues in the event of any future modifications